modules/ROOT/pages/afm-monitoring-public-apis.adoc

```diff
@@ -64,12 +64,16 @@ To prevent tests run in public locations from consuming too many resources, AFM
 
 == Scheduling Tests
 
-Tests are run at fixed intervals that you define in schedules. Each test is associated with a schedule. When you run tests against the endpoints of public APIs, AFM sets limitations on the following aspects of scheduling:
+Tests are run at fixed intervals that you define in schedules. Each test is associated with a schedule. When you run tests against the endpoints of public APIs, AFM sets limits on the following aspects of scheduling:
```

Comment:
> When you run tests against the endpoints of public APIs

Tests that are run on a public location has this limits. It does not depend on the API (because the customer can test a public API from a private location)

Reply:
Thanks, @Christian-Vallejos. I'll modify this as follows: 

When you run tests from public locations, ...